app/assets/javascripts/views/story_view.js

```diff
@@ -546,6 +546,7 @@ module.exports = FormView.extend({
 
       this.renderNotes();
 
+      this.renderReactComponents();
```

Comment:
Hmm, right. 

Well, I don't see what I said changes the primary intention suggested by @talyssonoc, but this approach won't be a problem too.

:+1: